src/components/ComponentPages/UploadFiles/UploadFilesUI/index.tsx

```diff
@@ -258,7 +259,12 @@ const UploadFileUI = ({
     return (
       <div>
         {(() => {
-          if (obj.type == "file" || obj.file_path) {
+          if (
+            (obj.file_type == "image/jpeg" ||
```

Comment:
This can replaced with regex

`^image\/(jpeg$|png$|jpg$)` and use this in if condition. It'll reduce the complexity

Reply:
done